sum_of_nth_terms.py

```diff
@@ -0,0 +1,16 @@
+'''My code for sum-of-nth-terms kata.'''
+
+
+def series_sum(n):
+    '''Add 1 divided by a div increasing by three for range n.'''
+    if n == 0:
+        return '0.00'
+    acc = 0
+    div = -2
+    for i in range(n):
+        div += 3
+        acc += 1 / div
```

Comment:
This is ok, but you don't need to carry so many separate variables. The denominator is always `1 + i * 3`, so you could just carry that forward for every `i` in `range(n)`